ReduxCore/inc/fields/typography/field_typography.js

```diff
@@ -349,20 +496,25 @@
         //console.log(color);
 
         //var output = family;
+        
+        //add
+        if(active){
 
         // Is selected font a google font?
         var google;
         if ( isSelecting === true ) {
-            google = redux.field_objects.typography.makeBool( selVals.object['data-google'] );
+            google = redux.field_objects.typography.makeBool( selVals['data-google'] );
             $( '#' + mainID + ' .redux-typography-google-font' ).val( google );
         } else {
             google = redux.field_objects.typography.makeBool( $( '#' + mainID + ' .redux-typography-google-font' ).val() ); // Check if font is a google font
         }
 
         // Page load. Speeds things up memory wise to offload to client
         if ( !$( '#' + mainID ).hasClass( 'typography-initialized' ) ) {
-            style = $( '#' + mainID + ' select.redux-typography-style' ).data( 'value' );
-            script = $( '#' + mainID + ' select.redux-typography-subsets' ).data( 'value' );
+            //style = $( '#' + mainID + ' select.redux-typography-style' ).data( 'value' );
+            //script = $( '#' + mainID + ' select.redux-typography-subsets' ).data( 'value' );
+            style = $( '#' + mainID + ' select.redux-typography-style' ).attr( 'data-value' );
+            script = $( '#' + mainID + ' select.redux-typography-subsets' ).attr( 'data-value' );
```

Comment:
I don't get this.  What's wrong with the .data function?  No need for .attr on data attributes.